crates/cubecl-cuda/src/compute/server.rs

```diff
@@ -644,7 +644,7 @@ impl CudaContext {
         let compute_kernel = kernel_compiled.repr.as_ref().unwrap();
         let cube_dim = kernel_compiled.cube_dim;
         let fast_math = compute_kernel.flags.inst_fast_math;
-        let arch = format!("--gpu-architecture=sm_{}", self.arch);
+        let arch = format!("--gpu-architecture=sm_{}a", self.arch);
```

Comment:
Why a ?

Reply:
So we can use architecture specific features like fp6. You did remind me to test on 3080 though and it seems like that's only supported as a tag for 90+. So I'll have to gate it. I was hoping they just added it as an alias to older arches for simplicity.

Reply:
They did add an alias for the SASS target even though it technically only means something for the PTX generation, just so you can use the same for both. But I guess that didn't apply for previous arches.